sphinx/source/docs/user_guide/tools.rst

```diff
@@ -812,7 +807,7 @@ list:
     r1 = p.rect('x', 'y', 0.1, 0.1, source=source2)
     tool = PointDrawTool(renderers=[c1, r1])
 
-The tool will automatically modify the columns on the data source
+The tool automatically modifies the columns on the data source
 corresponding to the ``x`` and ``y`` values of the glyph. Any
 additional columns in the data source will be padded with the declared
 ``empty_value``, when adding a new point. Any newly added points will
```

Comment:
regarding line 818: there is another "we" that should be replaced ("Using the ``num_objects`` property, we can ensure that")